src/Tizen.NUI/src/public/BaseComponents/ViewAccessibilityEvent.cs

```diff
@@ -0,0 +1,425 @@
+/*
+ * Copyright(c) 2021 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+using System;
+using System.ComponentModel;
+using System.Runtime.InteropServices;
+using Tizen.NUI.Components;
+using Tizen.NUI;
+
+namespace Tizen.NUI.BaseComponents
+{
+    public partial class View
+    {
+        internal class ControlHandle : SafeHandle {
+            public ControlHandle() : base (IntPtr.Zero, true) {}
+
+            public ControlHandle(IntPtr ptr) : base(ptr, true) {}
+
+            public override bool IsInvalid {
+                get {
+                    return this.handle == IntPtr.Zero;
+                }
+            }
+
+            protected override bool ReleaseHandle() {
+                Interop.View.DeleteControlHandleView(handle);
+                this.SetHandle(IntPtr.Zero);
+                return true;
+            }
+        }
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        ControlHandle GetControl() {
+            var ret = new ControlHandle(Interop.View.DownCast(SwigCPtr));
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            return ret;
+        }
+
+        ///////////////////////////////////////////////////////////////////
+        // ***************** AccessiblityDoGestureSignal ****************//
+        ///////////////////////////////////////////////////////////////////
+
+        private delegate void GestureInfoHandlerType(IntPtr data);
+        private GestureInfoHandlerType gestureInfoCallback;
+        private EventHandler<GestureInfoArgs> gestureInfoHandler;
+
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public struct GestureInfoType {
```

Comment:
Could you remove build warnings?

GestureInfoType should override Equals.
GestureInfoType should override the equality (==) and inequality (!=) operators.
Do not nest type GestureInfoType. Alternatively, change its accessibility so that it is not externally visible.

Reply:
Done.